Documentation/users/user-auth.md

```diff
@@ -0,0 +1,71 @@
+# Troubleshooting user authentication
+
+
+### Authorizing users
+
+Tectonic Identity uses email address for authorization across all identity provider platforms. For example, if *john.doe@example.org* is marked as an admin, admin access is given only if John Doe logs in with the same email address. If users attempt to log in with anything other than email, role binding policies are not applied.
+
+The following outlines an example LDAP configuration for the user *john.doe* and an associated ClusterRoleBinding.
+
+```
+john.doe, Users, 5866a86d3187bc712e435b35, example.org
+dn: uid=john.doe,ou=Users,o=5866a86d3187bc712e435b35,dc=example,dc=org
+givenName: john
+jcLdapAdmin: TRUE
+uid: john.doe
+uidNumber: 5006
+loginShell: /bin/bash
+homeDirectory: /home/John.doe
+sn: Doe
+cn: John Doe
+objectClass: exampleUser
+gidNumber: 5006
+mail: john.doe@example.org
+memberOf: cn=tectonic_users,ou=Users,o=5866a86d3187bc712e435b35,dc=example,dc=org
+```
+
+Use the email address as the name attribute for the account:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: admin-john.doe-namespace-all-access-binding
+  namespace: john.doe-namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: john.doe@example.org
+subjects:
+- kind: User
+  name: john.doe
+  namespace: john.doe-namespace
+```
+
+### Troubleshooting users and groups configuration
+
+To troubleshoot users and groups configuration, check the Tectonic Console's logs to see the LDAP query being sent.
```

Comment:
If its not too much redundancy maybe you can point them to where the Console logs are located.